frontend/public/components/start-guide.tsx

```diff
@@ -58,7 +63,7 @@ export const withStartGuide = (WrappedComponent, disable: boolean = true) =>
     const kind = _.get(kindObj, 'kind', rest.kind);
 
     // The start guide does not need to be shown on the Projects list page.
-    if (kind === ProjectModel.kind) {
+    if (CLUSTER_WIDE_WHITELIST.includes(kind)) {
```

Comment:
IMO, we still only need to hide the Getting Started message on the projects page. I think [this](https://github.com/openshift/console/blob/c8d960ac111224ccdf3570a0dcc6e56f1b3a5870/frontend/public/components/start-guide.tsx#L75) is where we need to adjust logic so that only namespaced resources get disabled.

```suggestion
    if (kind === ProjectModel.kind) {
```